Sources/SimpleNetworkService/SimpleNetworkService.swift

```diff
@@ -0,0 +1,8 @@
+/// A stub object for future use as a network layer.
+final public class SimpleNetworkService {
+    /// Custom initializer
+    public init() {}
```

Comment:
Does it really a custom initializer?
```suggestion
    /// Initialize the new instance of `SimpleNetworkService`
    public init() {}
```

Reply:
thx, fixed

Reply:
fixed